### PR TITLE
feat(graphql): add `toCreateRequestContent` field to contract types

### PR DESCRIPTION
**Pull request by kiurchv**

### CODE REVIEW

File: apps/graphql/features/contexts/context.exs [hunk 1]
```diff
@@ -162,6 +162,25 @@ defmodule GraphQL.Features.Context do
...
+    fn %{existing: existing} = state, %{entity_name: entity_name, table_data: table_data} ->
+      entity_name = Inflex.singularize(entity_name)
+      model = entity_name_to_model(entity_name)
+
+      table_data = transpose_table(table_data)
+      attrs = prepare_attrs(model, table_data)
+
+      item =
+        case entity_name_to_factory_args(entity_name) do
+          {nil, factory_name} -> build(factory_name, attrs)
```
Comment (progsmile) [inline apps/graphql/features/contexts/context.exs]: How about using some other value except `nil` to build entity? Nil means nothing, suggest `:build`

File: apps/graphql/features/contexts/context.exs [hunk 2]
```diff
@@ -2298,7 +2381,7 @@ defmodule GraphQL.Features.Context do
     post(conn, @graphql_path, %{query: query, variables: variables})
   end
 
-  @spec entity_name_to_model(entity_name :: string) :: module
+  @spec entity_name_to_model(entity_name :: String.t()) :: module
```
Comment (progsmile) [inline apps/graphql/features/contexts/context.exs]: BTW, term model is rarely used with ecto. Suggest to rename to `entity_name_to_schema`

File: apps/graphql/features/contexts/context.exs [hunk 3]
```diff
@@ -2346,6 +2433,10 @@ defmodule GraphQL.Features.Context do
   def entity_name_to_factory_args("settlement"), do: {nil, :settlement}
   def entity_name_to_factory_args(entity_name), do: raise("Factory not found for #{inspect(entity_name)}")
 
+  def transpose_table(table_data) do
+    for %{field: field, value: value} <- table_data, do: {field, value}, into: %{}
```
Comment (progsmile) [inline apps/graphql/features/contexts/context.exs]:
```suggestion
    Enum.into(table_data, %{}, &({&1.field, &1.value}))
```

File: apps/graphql/features/reimbursement_contracts/get_specific.feature [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      | "37b78300-4185-48f2-a1a7-a51aad8f9a6b" | "PHARMACY" | "e04303a1-3c62-4b45-9288-1859c65c815f" |
+    And the following employees exist:
+      | databaseId                             | employeeType | legalEntityId                          |
+      | "1bd6bbb4-2783-4ce5-841c-4461b69426d6" | "DOCTOR"     | "e04303a1-3c62-4b45-9288-1859c65c815f" |
+      | "d69fa719-5771-46aa-b07a-162abc6f155b" | "DOCTOR"     | "e04303a1-3c62-4b45-9288-1859c65c815f" |
+    And the following medical programs exist:
+      | databaseId                             |
+      | "7ffdd3b4-a3c7-40a9-bc62-4c2bdc65deb5" |
+    And a reimbursement contract with the following fields exist:
+      | field                    | value                                                                                                                                                                                                                                                                                                                                        |
```
Comment (progsmile) [inline apps/graphql/features/reimbursement_contracts/get_specific.feature]: remove extra spaces from table

File: apps/graphql/features/reimbursement_contracts/get_specific.feature [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      | nhsSignerID              | "65735584-b938-4882-bab5-207cf2bda235"                                                                                                                                                                                                                                                                                                       |
+      | startDate                | "2019-03-28"                                                                                                                                                                                                                                                                                                                                 |
+    And the following contract divisions exist:
+      | contractId                             | divisionId                             |
```
Comment (progsmile) [inline apps/graphql/features/reimbursement_contracts/get_specific.feature]: remove extra spaces from table

File: apps/graphql/features/reimbursement_contracts/get_specific.feature [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      | "ab9328f4-f9dc-4135-a126-2acc4ac4a1af" | "308375d9-37f6-4a82-a4ed-1d6a840f06de" |
+      | "ab9328f4-f9dc-4135-a126-2acc4ac4a1af" | "37b78300-4185-48f2-a1a7-a51aad8f9a6b" |
+    And my scope is "contract:read"
+    And my client type is "NHS"
+    And my client ID is "923f79db-4539-499f-be03-6d2a5d3d881a"
+    When I request toCreateRequestContent of the reimbursement contract where databaseId is "ab9328f4-f9dc-4135-a126-2acc4ac4a1af"
+    Then no errors should be returned
+    And I should receive requested item
+    And the toCreateRequestContent of the requested item should have the following fields:
+      | field                         | value                                                                                                                                                                                                                                                                                                                                        |
```
Comment (progsmile) [inline apps/graphql/features/reimbursement_contracts/get_specific.feature]: remove extra spaces from table

File: apps/graphql/features/reimbursement_contracts/get_specific.feature (hunk 4 above)
Comment (kiurchv) [reply apps/graphql/features/reimbursement_contracts/get_specific.feature]: Where there are extra spaces?

Comment (progsmile) [reply apps/graphql/features/reimbursement_contracts/get_specific.feature]: Right after `value` column. It makes table wide

File: apps/graphql/features/reimbursement_contracts/get_specific.feature [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+      | issueCity                | "Київ"                                                                                                                                                                                                                                                                                                                                       |
+      | medicalProgramId         | "7ffdd3b4-a3c7-40a9-bc62-4c2bdc65deb5"                                                                                                                                                                                                                                                                                                       |
+      | nhsLegalEntityID         | "923f79db-4539-499f-be03-6d2a5d3d881a"                                                                                                                                                                                                                                                                                                       |
```
Comment (progsmile) [inline apps/graphql/features/reimbursement_contracts/get_specific.feature]: Let's use id suffix consistently: `Id`, or `ID`

**Pull request merged**